torch/testing/_internal/common_methods_invocations.py

```diff
@@ -281,6 +281,31 @@ def sample_inputs_addmm(op_info, device, dtype, requires_grad):
                                     low=None, high=None,
                                     requires_grad=False))),)
 
+def sample_inputs_slogdet(op_info, device, dtype, requires_grad):
+    from torch.testing._internal.common_utils import random_hermitian_matrix, random_hermitian_pd_matrix
+
+    # tests cases are copied from 'method_tests'
+    test_inputs = (
```

Comment:
Does this handle empty tensors? That is, where one or more of the batch dimensions is zero, or where one of the matrices' dimensions is zero?

Reply:
`torch.linalg.slogdet` works for such inputs and I added them to the tests.